src/js/components/FormGroupComponent.jsx

```diff
@@ -53,7 +53,7 @@ var FormGroupComponent = React.createClass({
     var child = React.Children.only(props.children);
     var formControlChild = React.cloneElement(
       child, {
-        className: "form-control",
+        className: "form-control mousetrap",
```

Comment:
`.mousetrap` enables mousetrap (the key bindings) on the input fields where they are normally do not work. The class is needed to make the CMD+enter available.